indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -29,7 +29,7 @@
     </td>
 {%- endmacro %}
 
-{% macro render_abstract_list(abstracts, dynamic_columns, static_columns, total_abstracts, context_track=None) %}
+{% macro render_abstract_list(abstracts, dynamic_columns, static_columns, total_abstracts, context_track=None, reviewed_abstracts=None) %}
```

Comment:
`none` (also for `context_track`) - in jinja lowercase is preferred for none/true/false